tui/install.go

```diff
@@ -25,6 +30,26 @@ type InstallPage struct {
 	prgMax    int
 }
 
+// LangString hold strings for each translated language
+// and one for default if the current language is unavailable
+type LangString struct {
+	Default string `yaml:"default,omitempty"`
+	EnUs    string `yaml:"en_US,omitempty"`
+}
+
+// Messenger is an array of language string used to create
+// an end user message for display
+type Messenger struct {
+	Messages []*LangString `yaml:"install-msg,omitempty"`
+}
+
+const (
+	//installDataURLBase = "https://cdn.download.clearlinux.org/releases/28090/clear/config/image/.data"
+	installDataURLBase = "http://mhorn-linux.jf.intel.com/pre-release/.data/"
```

Comment:
"http://mhorn-linux.jf.intel.com/pre-release/.data/"

Reply:
Fixed this to the proper URL, not a test.